Installation-Static Build-Windows (#20122)
netdata/netdata: packaging/installer/methods/freebsd.md

# Install Netdata on FreeBSD

> > 💡 This guide is community-maintained and might not always reflect the latest details (like package versions).  

Reviewer: I feel we need to streamline these @ilyam8 @kanelatechnical 

```suggestion
> **Note** 
>
> This guide is community-maintained and might not always reflect the latest details (like package versions).  
```

| `--native-only`                                      | Install only if native packages are available.                                                             |
| `--static-only`                                      | Install only if static builds are available.                                                               |
| `--install-prefix /opt`                              | Change installation directory.                                                                             |
| `--prepare-offline-install-source ./netdata-offline` | Prepare offline installation source. See [Offline Install Guide](/packaging/installer/methods/offline.md). |

---

## Environment Variables (Advanced Users)

| Variable              | Purpose                                                     |
|-----------------------|-------------------------------------------------------------|
| `TMPDIR`              | Directory for temporary files.                              |
| `ROOTCMD`             | Command used for privilege escalation (default: sudo/doas). |
| `DISABLE_TELEMETRY=1` | Disable anonymous telemetry data.                           |

---

> ## Telemetry Notice 📊

Reviewer: note for self, for review, this might look odd in TOC

> ## Telemetry Notice 📊

Reviewer: ```suggestion
## Telemetry Notice
```

This is the rendering when you deploy a title w/ emojis on Learn, I don't particularly like it, thus I suggest not using them. If you fashion it though, I don't have a strong objection against it, just personal preference

![image](https://github.com/user-attachments/assets/0ddf4aca-8b87-4d74-9386-d3824fca1398)

> Starting with Netdata v1.30, anonymous usage data is collected by default.
> 
> Learn more or opt-out:  
> [Anonymous Telemetry Events](/docs/netdata-agent/configuration/anonymous-telemetry-events.md)

Reviewer: we are on 2.4.0, so I would nuke this version notice

```suggestion
Anonymous usage data is collected by default. You can learn more or opt-out [here](/docs/netdata-agent/configuration/anonymous-telemetry-events.md)
```
